src/app/provider/overlay-provider.tsx

```diff
@@ -0,0 +1,7 @@
+'use client';
+
+import { OverlayProvider } from 'overlay-kit';
+
+export default function ClientOverlayProvider({ children }: { children: React.ReactNode }) {
```

Comment:
따로 Props Interface를 따로 정의하는게 어떨까요?


```suggestion
import { ReactNode } from 'react';

interface Props {
    children: ReactNode;
}

export default function ClientOverlayProvider({ children }: Props) {
```